ledger/accountdb.go

```diff
@@ -5364,6 +5364,22 @@ func stateProofVerificationInitDbQueries(r db.Queryable) (*stateProofVerificatio
 	return qs, nil
 }
 
+func getLatestVerificationData(handle *sql.DB) (ledgercore.StateProofVerificationData, error) {
```

Comment:
I'm pretty sure the data we need belongs to the earliest round, doesn't it? The later data won't be used as long as earlier data exists to verify earlier state proofs.

Reply:
As far as I understand, the task is to ensure quick access to `trackedCommitData[len(trackedCommitData)-1]` in the  "everything is okay" case.
That is,  ensuring that after flushing to the DB, the access to the last-seen SP verification data is in memory and can be quickly retrieved.